nullarbor/2.0.20191013/Dockerfile

```diff
@@ -0,0 +1,395 @@
+# roary via ubuntu:focal apt is 3.13.0
+RUN apt-get update && apt-get install -y --no-install-recommends \
+ git \
+ python \
+ wget \
+ pigz \
+ zlib1g-dev \
+ make \
+ gcc \
+ g++ \
+ libpthread-stubs0-dev \
+ openjdk-11-jre \
+ unzip \
+ bzip2 \
+ libncurses5-dev \
+ libbz2-dev \
+ liblzma-dev \
+ libcurl4-gnutls-dev \
+ libssl-dev \
+ libfindbin-libs-perl \
+ libyaml-tiny-perl \
+ libautobox-list-util-perl \
+ bzip2 \
+ gzip \
+ libdatetime-perl \
+ libxml-simple-perl \
+ libdigest-md5-perl \
+ bioperl \
+ hmmer \
+ libpath-tiny-perl \
+ libmoo-perl \
+ libfile-which-perl \
+ libsvg-perl \
+ libtext-csv-perl \
+ liblist-moreutils-perl \
+ libio-file-withpath-perl \
+ roary=3.13.0+dfsg-1 \
+ libmoo-perl \
+ liblist-moreutils-perl \
+ libjson-perl \
+ file \
+ ncbi-blast+=2.9.0-2 \
+ autoconf \
+ automake \
+ libtool \
+ flex \
+ bison && \
+ apt-get clean && apt-get autoclean && \
+ rm -rf /var/lib/apt/lists/*
+
+# SPAdes
+RUN wget https://github.com/ablab/spades/releases/download/v${SPADES_VER}/SPAdes-${SPADES_VER}-Linux.tar.gz && \
+  tar -xzf SPAdes-${SPADES_VER}-Linux.tar.gz && \
+  rm SPAdes-${SPADES_VER}-Linux.tar.gz
+
+# Seqtk 1.3
+RUN wget https://github.com/lh3/seqtk/archive/v1.3.tar.gz && \
+  tar -zxf v1.3.tar.gz && \
+  rm v1.3.tar.gz && \
+  cd seqtk-1.3/ && \
+  make && \
+  make install
+
+# kmc
+RUN mkdir kmc && \
+  cd kmc && \
+  wget https://github.com/refresh-bio/KMC/releases/download/v3.1.1/KMC3.1.1.linux.tar.gz && \
+  tar -xzf KMC3.1.1.linux.tar.gz && \
+  rm KMC3.1.1.linux.tar.gz  
+
+# lighter 1.1.1
+RUN wget https://github.com/mourisl/Lighter/archive/v1.1.1.tar.gz && \
+  tar -zxf v1.1.1.tar.gz && \
+  rm -rf v1.1.1.tar.gz && \
+  cd Lighter-1.1.1 && \
+  make
+
+# trimmomatic
+RUN wget http://www.usadellab.org/cms/uploads/supplementary/Trimmomatic/Trimmomatic-${TRIMMOMATIC_VER}.zip && \
+  unzip Trimmomatic-${TRIMMOMATIC_VER}.zip && \
+  rm -rf Trimmomatic-${TRIMMOMATIC_VER}.zip && \
+  chmod +x Trimmomatic-${TRIMMOMATIC_VER}/trimmomatic-${TRIMMOMATIC_VER}.jar && \
+  echo "#!/bin/bash" >> trimmomatic && \
+  echo "exec java -jar /Trimmomatic-${TRIMMOMATIC_VER}/trimmomatic-${TRIMMOMATIC_VER}.jar """"$""@"""" " >> trimmomatic && \
+  chmod +x trimmomatic && \
+  mv -v trimmomatic /usr/local/bin
+
+# bwa (mem) 0.7.17
+RUN wget https://github.com/lh3/bwa/releases/download/v0.7.17/bwa-0.7.17.tar.bz2 && \
+  tar -xjf bwa-0.7.17.tar.bz2 && \
+  rm bwa-0.7.17.tar.bz2 && \
+  cd bwa-0.7.17 && \
+  make
+
+# samtools 1.10 
+RUN mkdir samtools && \
+  cd samtools && \
+  wget https://github.com/samtools/samtools/releases/download/1.10/samtools-1.10.tar.bz2 && \
+  tar -xjf samtools-1.10.tar.bz2 && \
+  rm samtools-1.10.tar.bz2 && \
+  cd samtools-1.10 && \
+  ./configure && \
+  make && \
+  make install
+
+# skesa 2.3.0 (skesa 2.4.0 binary works better on ubuntu:bionic, so not upgrading here)
+RUN mkdir skesa && \
+  cd skesa && \
+  wget https://github.com/ncbi/SKESA/releases/download/v${SKESA_VER}/skesa.centos6.10 && \
```

Comment:
This `wget` link needs updating. It should be: `https://github.com/ncbi/SKESA/releases/download/${SKESA_VER}/skesa.centos.7.7` Then update the move command with the new filename `mv -v skesa.centos.7.7 skesa`